docs/edgekv_cli.md

```diff
@@ -225,6 +225,7 @@ Usage: `akamai edgekv write <itemType> <environment> <namespace> <groupId> <item
 | Option | Description |
 | - | - |
 | -h, --help  | Display information on how to use this EdgeKV command |
+| --sandboxid| optional |`sandbox-id` to use for the data operation. You can use the `akamai sandbox list` CLI command to view a list of available sandboxes.|
```

Comment:
can you please have it as "sandboxId". Since all the other options are camelcase we can keep it consistent.